docs/general/builtins/scalar-functions.rst

```diff
@@ -4438,6 +4438,29 @@ An example::
     SELECT 2 rows in set (... sec)
 
 
+.. _scalar_vector_similarity:
+
+
+``vector_similarity(float_vector, float_vector, float)``
```

Comment:
I think this scalar should be placed under a new `vector` section.

Reply:
Done in https://github.com/crate/crate/pull/15832/commits/2f5dddf244b28321fd2b792ba1cd59fc39ab99fd.

Should I move `knn_match` there as well? It's currently in the "special functions" section.